Essentials/src/com/earth2me/essentials/services/ReserveEssentials.java

```diff
@@ -0,0 +1,966 @@
+        }
+        return ess.getSettings().getStartingBalance();
+    }
+
+    /**
+     * Used to get the balance of an account.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @return The balance of the account.
+     */
+    @Override
+    public BigDecimal getHoldings(UUID identifier) {
+        if (hasAccount(identifier)) {
+            try {
+                return Economy.getMoneyExact(getName(identifier));
+            } catch (UserDoesNotExistException ignore) { }
+        }
+        return ess.getSettings().getStartingBalance();
+    }
+
+    /**
+     * Used to get the balance of an account.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param world The name of the {@link World} associated with the balance.
+     * @return The balance of the account.
+     */
+    @Override
+    public BigDecimal getHoldings(String identifier, String world) {
+        return getHoldings(identifier);
+    }
+
+    /**
+     * Used to get the balance of an account.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param world The name of the {@link World} associated with the balance.
+     * @return The balance of the account.
+     */
+    @Override
+    public BigDecimal getHoldings(UUID identifier, String world) {
+        return getHoldings(identifier);
+    }
+
+    /**
+     * Used to get the balance of an account.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param world The name of the {@link World} associated with the balance.
+     * @param currency The {@link Currency} associated with the balance.
+     * @return The balance of the account.
+     */
+    @Override
+    public BigDecimal getHoldings(String identifier, String world, String currency) {
+        return getHoldings(identifier);
+    }
+
+    /**
+     * Used to get the balance of an account.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param world The name of the {@link World} associated with the balance.
+     * @param currency The {@link Currency} associated with the balance.
+     * @return The balance of the account.
+     */
+    @Override
+    public BigDecimal getHoldings(UUID identifier, String world, String currency) {
+        return getHoldings(identifier);
+    }
+
+    /**
+     * Used to determine if an account has at least an amount of funds.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param amount The amount you wish to use for this check.
+     * @return True if the account has at least the specified amount of funds, otherwise false.
+     */
+    @Override
+    public boolean hasHoldings(String identifier, BigDecimal amount) {
+        try {
+            return hasAccount(identifier) && Economy.hasEnough(identifier, amount);
```

Comment:
Given the exception catch below, should probably just be Economy.hasEnough and let it throw.